### ISSUE TITLE
How to read the .dat files?

### ISSUE THREAD

**Issue by e1iset** [lululxvi/deep-learning-for-indentation#5]
Thanks for sharing this novel approach to study the material properties. I'm a beginner at the neural network and have some stupid questions about how to read the .dat files outputted from nn.py code.
1) In test.dat and train.dat, why the x values are not the same as the ones in the .csv files?
2) There are 2 columns of y values, as well as two columns of predicted y values, what's the purpose of having two columns of the same type of data?
3) In the experiment data, the elastoplastic properties are measured from each test or referred from a tenable value of the same material?

**Comment by lululxvi** [comment]
1. The data is normalized for training.
2. One is from low-fidelity NN, and the other one is from high-fidelity NN.
3. The properties are measured from each material.

**Comment by e1iset** [comment]
> 1. The data is normalized for training.
> 2. One is from low-fidelity NN, and the other one is from high-fidelity NN.
> 3. The properties are measured from each material.

Thanks!
:)

**Comment by LanPeng-94** [comment]
> 1. The data is normalized for training.
> 2. One is from low-fidelity NN, and the other one is from high-fidelity NN.
> 3. The properties are measured from each material.

Hi Lu,

We use MFNN to train my datasets. The inputs of NN are x1, x2, and x3, and the output of NN is y. 
In test.dat, I find 7 columns, and I know the first 3 columns are input x1, x2, and x3, but I'm not sure what the last 4 columns represent.

Why are columns 4 and 5 the same, low fidelity NN output first or high fidelity NN output first?

![Image](https://github.com/user-attachments/assets/73042215-0046-422f-89f2-dd6a7caa4d63)

**Comment by lululxvi** [comment]
This is because for the test data, there is no low-fidelity ground truth provided; then the code simply copys the high fidelity as the low fidelity. You can ignore that.
